src/tutorials/pet.md

```diff
@@ -198,16 +322,11 @@ NewBidsDataSet/
 machine:Projects user$
 ```
 
-That certainly looks less daunting, now let's change the filenames of the
-templates so that they make more sense for our data set (aka remove ShortExample
-from each filename).
+That certainly looks less daunting, now let's change the filenames of the templates so that they make more sense for our data set (aka remove ShortExample from each filename).
 
-**Note:** if you have multiple PET image files you can distinguish between them
-by creating a session folder with a unique name and then applying the
-`ses-<label>` label to each file therein. If there's a single pet scan you may
-omit the additional folder and corresponding label(s) from the filename. For
-more information on labeling see
-[this link](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/09-positron-emission-tomography.html#pet-recording-data).
+**Note:** if you have multiple PET image files you can distinguish between them by creating a session folder with a unique name and then applying the
+`ses-<label>` label to each file therein. If there's a single pet scan you may omit the additional folder and corresponding label(s) from the filename. For
+more information on labeling see [this link](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/09-positron-emission-tomography.html#pet-recording-data).
```

Comment:
```suggestion
```{Note}
If you have multiple PET image files you can distinguish between them by creating a session folder with a unique name
and then applying the `ses-<label>` label to each file therein. 
If there's a single pet scan you may omit the additional folder and corresponding label(s) from the filename.
For more information on labeling see [this link](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/09-positron-emission-tomography.html#pet-recording-data).
```
```